src/middlewared/middlewared/plugins/system_advanced/config.py

```diff
@@ -297,6 +297,9 @@ async def do_update(self, data):
 
             await self.middleware.call('system.advanced.configure_tty', original_data, config_data, generate_grub)
 
+            if config_data['debugkernel']:
```

Comment:
Just wondering why we don't **only** do this if we've _just_ turned on debugkernel, e.g.
```
            if config_data['debugkernel'] and not original_data['debugkernel']:
                await self.middleware.call('boot.update_initramfs')
```